src/Common/src/Interop/Interop.UnsafeNativeMethods.Odbc.cs

```diff
@@ -1,15 +1,16 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
 using System.Data.Odbc;
 using System.Runtime.ConstrainedExecution;
 using System.Runtime.InteropServices;
 using System.Runtime.Versioning;
 using System.Security;
 using System.Text;
 
-namespace System.Data.Common
+internal static partial class Interop
 {
     [SuppressUnmanagedCodeSecurity]
     internal static partial class UnsafeNativeMethods
```

Comment:
Following the guidance at https://github.com/dotnet/corefx/blob/42ce0e696e08e190aceaf8613e47e67d57178550/Documentation/coding-guidelines/interop-guidelines.md#interop-type and how it's done for other types like Interop.Kernel32, Interop.Winsock, Interop.Crypto, etc., this should be Interop.Odbc or Odbc32.